src/components/pages/Contacts/index.tsx

```diff
@@ -117,6 +116,20 @@ const UnconnectedContacts: React.FC<PropsFromRedux> = ({
     page_id = 'faded';
   }
 
+  const handleScroll = (e: UIEvent) => {
+    const tableDiv = document.getElementById('tableDiv');
+    // const org = user.user.org;
+
+    if (tableDiv) {
+      const isBottom =
```

Comment:
maybe we can create a isBottom function on the util file? Seems like a useful function to have around!

Reply:
yeah that sounds good - put into utils